nilearn/image/resampling.py

```diff
@@ -287,7 +288,7 @@ def _resample_one_img(data, A, b, target_shape,
 
 def resample_img(img, target_affine=None, target_shape=None,
                  interpolation='continuous', copy=True, order="F",
-                 clip=True):
+                 clip=True, fill=0):
```

Comment:
I would prefer this parameter to be called "fill_value"

Reply:
sure.